src/models/layerMetadata/layerMetadata.ts

```diff
@@ -1064,6 +1064,51 @@ export class LayerMetadata implements ILayerMetadata, IMetadataCommonModel {
   //#endregion
   public displayPath: string | undefined = undefined;
 
+  //#region RASTER: transparency
+  @pycsw({
+    profile: 'mc_raster',
+    xmlElement: 'mc:transparency',
+    queryableField: 'mc:transparency',
+    pycswField: 'pycsw:Transparency',
```

Comment:
why do you use capital "T" letter?

Reply:
copy the convention from other examples of types

Reply:
changed + on pycsw